test/parallel/test-zlib-brotli.js

```diff
@@ -66,8 +66,8 @@ const sampleBuffer = fixtures.readSync('/pss-vectors.json');
       }
     });
   }, {
-    code: 'ERR_ZLIB_INITIALIZATION_FAILED',
-    name: 'Error',
-    message: 'Initialization failed'
+    code: 'ERR_INVALID_CALLBACK',
+    name: 'TypeError',
+    message: 'Invalid Async Callback Function type'
```

Comment:
If this is changing the error that’s being emitted then this is introducing a bug. It might be better to return `undefined` instead of throwing an exception, then – that also keeps the `MaybeLocal<>` contract fulfilled.

Reply:
@addaleax - do you mean to suggest 

return `undefined` instead of the current `return v8::MaybeLocal<v8::Value>();` , 
or

detect this scenario and return `undefined` from the caller method (of `AsyncWrap::MakeCallback`) ?

Reply:
@HarshithaKP I’m referring to the former, yes

Reply:
@addaleax, thanks. Made suggested change and reverted changes made in `test/parallel/test-zlib-brotli.js`  and `src/node_errors.h`.